src/Forms/FixCommonErrors.cs

```diff
@@ -2207,27 +2207,27 @@ public static string FixStartWithUppercaseLetterAfterParagraph(Paragraph p, Para
 
             if (p.Text != null && p.Text.Contains(Environment.NewLine))
             {
-                string[] arr = p.Text.Replace("\r\n", "\n").Replace("\r", "\n").Split('\n');
+                var arr = p.Text.Split(Utilities.NewLineChars, StringSplitOptions.RemoveEmptyEntries);

```

Comment:
Yes, this is buggy - the split does not work as it removes empty entries... a new extension method "SplitToLines" could be made so avoid duplicate code for line splitting